app/views/renters/index.html.erb

```diff
@@ -1,7 +1,14 @@
 <h1>Renters</h1>
 
+<h4>Average Cost Per Sq Foot: <%= Unit.avg_cost_per_sq_ft %></h4>
```

Comment:
Avoid calling models directly in your view. Instead pass the instance variable holding all of the units, and then call this method on it. Looks like you're already passing `@units`, so you should be able to use `@units.avg_cost_per_sq_ft` here.